packages/workflow/src/WorkflowHooks.ts

```diff
@@ -25,12 +32,12 @@ export class WorkflowHooks {
 		this.retryOf = optionalParameters.retryOf;
 	}
 
-	async executeHookFunctions(hookName: string, parameters: any[]) { // tslint:disable-line:no-any
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Any chance we can specify a rule to allow for explict any when used in `any[]`? This type seems to require exceptions frequently.

Reply:
yeah I guess the explicit any is the thoughest decision we have to make 🙈 I understand both sides, accordingly I would love to get more opinions on it 👍🏼